app/controllers/metrics_controller.rb

```diff
@@ -0,0 +1,42 @@
+class MetricsController < ActionController::Base
+  def show
+    response.set_header('Content-Type', 'text/plain; version=0.0.4')
+    @stats = delayed_jobs_stats
+
+    render 'metrics/show.text'
+  end
+
+  private
+
+  def delayed_jobs_stats
+    [
+      success_stats,
+      pending_stats,
+      failed_stats
+    ]
+  end
+
+  def success_stats
+    {
+      name: :delayed_jobs_successful,
+      docstring: 'Number of successful jobs',
+      value: ProcessedSubmission.count
+    }
+  end
+
+  def pending_stats
+    {
+      name: :delayed_jobs_pending,
+      docstring: 'Number of pending jobs',
+      value: Delayed::Job.where('attempts = 0').count
+    }
+  end
+
+  def failed_stats
+    {
+      name: :delayed_jobs_failed,
+      docstring: 'Number of jobs failed',
+      value: Delayed::Job.where('attempts > 0').count
+    }
+  end
```

Comment:
The business logic should be pulled out of the controller. Will make the tests easier for metrics